foundationdb/manifest.json

```diff
@@ -0,0 +1,170 @@
+        "foundationdb.process.role.kvstore_available_bytes",
+        "foundationdb.process.role.kvstore_free_bytes",
+        "foundationdb.process.role.kvstore_inline_keys",
+        "foundationdb.process.role.kvstore_total_bytes",
+        "foundationdb.process.role.kvstore_total_nodes",
+        "foundationdb.process.role.kvstore_total_size",
+        "foundationdb.process.role.kvstore_used_bytes",
+        "foundationdb.process.cpu.usage_cores",
+        "foundationdb.process.disk.free_bytes",
+        "foundationdb.process.disk.reads.hz",
+        "foundationdb.process.disk.total_bytes",
+        "foundationdb.process.disk.writes.hz",
+        "foundationdb.process.memory.available_bytes",
+        "foundationdb.process.memory.limit_bytes",
+        "foundationdb.process.memory.unused_allocated_memory",
+        "foundationdb.process.memory.used_bytes",
+        "foundationdb.process.network.connection_errors.hz",
+        "foundationdb.process.network.connections_closed.hz",
+        "foundationdb.process.network.connections_established.hz",
+        "foundationdb.process.network.current_connections",
+        "foundationdb.process.network.megabits_received.hz",
+        "foundationdb.process.network.megabits_sent.hz",
+        "foundationdb.process.network.tls_policy_failures.hz",
+        "foundationdb.process.role.commit_latency_statistics.count",
+        "foundationdb.process.role.commit_latency_statistics.max",
+        "foundationdb.process.role.commit_latency_statistics.min",
+        "foundationdb.process.role.commit_latency_statistics.p25",
+        "foundationdb.process.role.commit_latency_statistics.p90",
+        "foundationdb.process.role.commit_latency_statistics.p99",
+        "foundationdb.process.role.data_lag.seconds",
+        "foundationdb.process.role.durability_lag.seconds",
+        "foundationdb.process.role.grv_latency_statistics.default.count",
+        "foundationdb.process.role.grv_latency_statistics.default.max",
+        "foundationdb.process.role.grv_latency_statistics.default.min",
+        "foundationdb.process.role.grv_latency_statistics.default.p25",
+        "foundationdb.process.role.grv_latency_statistics.default.p90",
+        "foundationdb.process.role.grv_latency_statistics.default.p99",
+        "foundationdb.process.role.read_latency_statistics.count",
+        "foundationdb.process.role.read_latency_statistics.max",
+        "foundationdb.process.role.read_latency_statistics.min",
+        "foundationdb.process.role.read_latency_statistics.p25",
+        "foundationdb.process.role.read_latency_statistics.p90",
+        "foundationdb.process.role.read_latency_statistics.p99"
+    ],
+  "creates_events": false,
+  "short_description": "FoundationDB integration",
+  "guid": "ed46c9de-3a62-46f0-98a6-19344746dd42",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "FoundationDB",
+  "categories": [
```

Comment:
```suggestion
  "categories": [
    "data store",
  ]
```

Reply:
Applied, though without the trailing comma (sigh, JSON :-)).